command/src/command.proto

```diff
@@ -151,7 +151,7 @@ message HttpsListenerConfig {
     required uint32 request_timeout = 10 [default = 10];
     // wether the listener is actively listening on its socket
     required bool active = 11 [default = false];
-    // TLS versions
+    // TLS versions
```

Comment:
Could you ensure that is not a `&nbsp;`

Reply:
I just did a check on the whole repository and found one non-breakable space in the protobuf file, and corrected it. Otherwise there was none.